react/features/filmstrip/actions.web.js

```diff
@@ -26,22 +25,7 @@ import {
     calculateThumbnailSizeForVerticalView
 } from './functions';
 
-/**
- * Sets the list of the reordered remote participants based on which the visible participants in the filmstrip will be
- * determined.
- *
- * @param {Array<string>} participants - The list of the remote participant endpoint IDs.
- * @returns {{
-        type: SET_REMOTE_PARTICIPANTS,
-        participants: Array<string>
-    }}
- */
-export function setRemoteParticipants(participants: Array<string>) {
-    return {
-        type: SET_REMOTE_PARTICIPANTS,
-        participants
-    };
-}
+export * from './actions.any';
```

Comment:
Now that we have actions.any.js, do we need this [line](https://github.com/jitsi/jitsi-meet/blob/master/react/features/filmstrip/actions.web.js#L196) still ?

Reply:
Yes. Without it we won't export the common actions in the web version.

Reply:
I was referring to https://github.com/jitsi/jitsi-meet/blob/master/react/features/filmstrip/actions.web.js#L196 where the actions.native are re-exported from actions.web.

Reply:
I saw the problem. will fix it